src/main/java/com/google/api/codegen/transformer/py/PythonGapicSurfaceTransformer.java

```diff
@@ -170,13 +171,21 @@ public PythonGapicSurfaceTransformer(
     }
 
     GrpcDocView enumFile =
-        generateEnumView(productConfig, modelTypeTable, namer, apiModel.getProtoModel().getFiles());
+        generateEnumView(productConfig, modelTypeTable, namer, getProtoFiles(productConfig));
     if (!enumFile.elementDocs().isEmpty()) {
       serviceSurfaces.add(enumFile);
     }
     return serviceSurfaces.build();
   }
 
+  private List<ProtoFile> getProtoFiles(GapicProductConfig productConfig) {
+    ImmutableList.Builder<ProtoFile> files = ImmutableList.builder();
+    for (ProtoFile protoFile : ProtoFiles.getProtoFiles(productConfig)) {
```

Comment:
Can this method just be `files = ImmutableList.copyOf(ProtoFiles.getProtoFiles(productConfig))`?

Reply:
If that is more idiomatic, sure.  My familiarity with Java is some years out of date at this point.